app/views/organisations/_organisation.html.slim

```diff
@@ -0,0 +1,119 @@
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    div class="govuk-!-margin-bottom-8"
+      h1.govuk-heading-l class="govuk-!-margin-bottom-1"
+        = organisation.name
+      p.govuk-body-m = full_address(organisation)
+
+- if !preview && organisation.school_group?
+  = tabs do |tabs|
```

Comment:
you need a `html_attributes: { "aria-label": "main menu" }` here as per prototype